ct_optcon/test/solver/linear/ManifoldLQOCTest.cpp

```diff
@@ -0,0 +1,514 @@
+
```

Comment:
abstract the composite manifold classes, move them to an appropriate place

Reply:
also check back on the latest developments within manif regarding composite manifolds.

Reply:
extend the generic NLOC classes with the novel features and make this a unit test which compares them against the solution below.